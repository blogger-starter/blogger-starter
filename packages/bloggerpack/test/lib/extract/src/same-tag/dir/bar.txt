<example>
bar
</end>

<example>
bar2
</end>

<example>
bar3
</end>
