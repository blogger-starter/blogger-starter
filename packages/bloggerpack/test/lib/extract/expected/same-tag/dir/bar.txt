/* src/same-tag/dir/bar.txt */

bar
